tropohelper/services.py

```diff
@@ -57,9 +57,13 @@ def create_json_redshift_firehose_from_stream(stack, name, firehose_arn,
                                               s3_bucket_arn, s3_kms_key_arn, s3_role_arn,
                                               s3_buffering_seconds=300, s3_buffering_size=5,
                                               s3_compression_format='GZIP',
-                                              s3_log_group_name='firehose-streams',
-                                              redshift_log_group_name='redshift-firehose'):
+                                              s3_log_group_name='firehose-s3',
+                                              redshift_log_group_name='firehose-redshift'):
     """Add Kinesus Redshift Firehose Resource with another Kinesis Stream as source and json as payload."""
+
+    s3_log_group = create_log_group(stack, s3_log_group_name);
```

Comment:
You don't need to create a log group for both S3 and Redshift, a single log group will suffice .

Reply:
done